disnix/support.tt

```diff
@@ -8,8 +8,7 @@
   href="irc://irc.freenode.net/#nixos"><tt>#nixos</tt></a> on <a
   href="https://freenode.net/"><tt>irc.freenode.net</tt></a>.</li>
 
-  <li>You can use the <strong>mailing list</strong> <a
-  href="https://groups.google.com/forum/#!forum/nix-devel"><tt>nix-devel</tt></a>.</li>
+  <li>You can use the <a href="[%root%]nixos/support.html#mailing-list"><tt>Discourse</tt> mailing list</a>.</li>
```

Comment:
In several places this PR refers to Discourse as a mailing list. Why not just "Discourse" or the "Nix community forum"? Just because it has a mail gateway doesn't make it a mailing list.

Reply:
Okay. I changed all references to "Discourse" or "Discourse forum" in 76a4f50159e3ae9694692e3b377a4d0cc5ccfe77